api/clients/google_workspace_client.py

```
@@ -329,42 +365,46 @@ def _calculate_users_quality(self, total_users: int, mfa_enabled: int,
+            if (
+                datetime.now(timezone.utc)
+                - datetime.fromisoformat(act.get("id", {}).get("time", "").replace("Z", "+00:00"))
+            ).days
+            < MAX_RETRIES
+        )
         if recent_events > 50:
             return EvidenceQuality.HIGH
         elif recent_events > 10:
             return EvidenceQuality.MEDIUM
         else:
             return EvidenceQuality.LOW
```

Pylint (3.3.8)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)